### ISSUE TITLE
Add a prominent "I'm a librarian, why should I care?" section

### ISSUE THREAD

**Issue by jezcope** [LibraryCarpentry/librarycarpentry.github.io#7]
From a recent email exchange:

> "One comment was that on the webpage things aren't explained entirely in layman's terms - we did need to resort to our semi-techie people for an explanation"

To get buy-in from potential learners (as opposed to instructors), I think we need to address this somewhere on either the home or about pages. Both of these currently go straight into detail about which tools are taught (git, OR, etc.) without explaining in lay terms why those skills are useful.

**Comment by drjwbaker** [comment]
> Library Carpentry is by librarians for librarians. Library Carpentry modules help you automate boring tasks, create and maintain sustainable and reusable data, work closely with IT professionals, and much more. Library Carpentry introduces you to the basics of computing and provides you with a platform for self-learning.

**Comment by weaverbel** [comment]
@jezcope @drjwbaker I also think we need to make the case for why digital skills are important in the wider picture, i.e. something along the lines of 'The 21st century is the century of data. In many cases now, researchers don't want librarians to find new information sources for them - they want help and guidance to find information within the data they already have - by mining it, analysing it and visualising it. By learning the tools ourselves, we can partner with our researchers to help them perform 21st century research.'

What do you think?

**Comment by richyvk** [comment]
Might want to mention that these skills are (potentially at least) wider than research support too. I liked @jezcope 's rewrite of the intro - the stuff about automating boring stuff etc. I think it's worth saying that these tools will help librarians tackle all sort of challenges, be they assisting researchers with thier data, or processing their own data, or indeed other things. Particularly if (when) LC adds more content - like web scraping, and perhaps things like - how to build a site for your library/project on GitHub pages with Jekyll. etc etc.

**Comment by drjwbaker** [comment]
I don't want to focus too much on research support because I'm uncomfortable with that coming across as the prime motivation. That said another bullet like..

> better understand software used in research

.. works for me.

**Comment by weaverbel** [comment]
Fine with me - and the research thing might put off librarians in public libraries, many of whom are interested here.

**Comment by jezcope** [comment]
Yes, I tend to talk about the research thing quite a bit because it's a good selling point in my own workplace, but I don't see it as the most important or valuable thing by any means.

**Comment by drjwbaker** [comment]
We just need to strike the right balance: research collaboration or research support isn't an in for everyone. I'm just conscious - as a researcher - not to overplay that hand.

**Comment by ccronje** [comment]
To librarians, I've described computational approaches as automating boring, repetitive tasks;
exploring and transforming messy files quickly and efficiently; discovering patterns in data e.g. extracting meaning for better decision-making, but also exploring creative thinking.

I explain that computational thinking starts to become part of your daily library practice e.g. simple but significant decisions like creating csv files where multi-tabbed proprietary spreadsheets are unnecessary, and 'computer friendly' file naming practice (as per the Data-Intro w/shop).

I showed a group of librarians the [LC good principles](https://github.com/data-lessons/librarycarpentry/issues/11) and it was interesting to hear them say they want more confidence to say what they do/don't know and get away from the unhelpful 'expert' characterisation that comes with the job. They seek to embody the freer more creative persona of 'the hacker'.

**Comment by drjwbaker** [comment]
Feeling confident to admit not knowing something seems super important. Perhaps suggest an edit/addition at https://github.com/LibraryCarpentry/librarycarpentry.github.io ?

**Comment by ccronje** [comment]
The desire to tinker, experiment and hack is strong in the library community. These form part of the learning/skills outcomes of LC and would be good to highlight. It would actually be fun to teach them about the commandline on a RaspberryPi to help boost their confidence and facilitate experimentation (it would also provide a workaround for locked down systems!).

**Comment by weaverbel** [comment]
@ccronje @drjwbaker So many locked down systems ! And it really crushes the urge to experiment. I tell all my people I am using a Linux machine I bought for $250. It had Windows on it which I threw away and installed Ubuntu desktop instead. When I show them how fast it is to boot up, how the battery goes all day even with heavy usage and how fast it is to update, they often think about getting one. There are lots of cheap or secondhand laptops out there (some as low as $50) and it really helps the learning curve if you can practise on your own machine. of course people with a Mac have a good terminal right there - it is Windows that is lame.

**Comment by gillianelliot** [comment]
Reflecting back to the very first post:
To get buy-in from potential learners (as opposed to instructors), I think we need to address this somewhere on either the home or about pages. Both of these currently go straight into detail about which tools are taught (git, OR, etc.) without explaining in lay terms why those skills are useful.

Yes, yes and yes. Please can the 'why bother' be clearly prioritised in the locations suggested. And I would personally find it very useful to read a brief case study at the start of each unit (real people real world problems/challenges). Listing what the tool does and how it does so seems a little dry to me.

**Comment by weaverbel** [comment]
All good points @gillianelliot - what did you think of my comments above? Useful/not useful?

**Comment by gillianelliot** [comment]
Do you mean your comments " I also think we need to make the case for..." if so, I would certainly agree the need for data mining, etc support - although I don't think the information support need has entirely disappeared. It doesn't help this sprint but we're going to probe our own librarians next week for what challenges they face, as a primer for LC. It may be that some of these challenges in turn could be used as real life (anonymised) case studies which respond to the 'why bother'
